src/Blocks/Progress.purs

```diff
@@ -0,0 +1,27 @@
+module Ocelot.Block.Progress where
+
+import Prelude
+
+import DOM.HTML.Indexed (HTMLspan, HTMLdiv)
+import Halogen.HTML as HH
+import Halogen.HTML.Properties as HP
+import Ocelot.HTML.Properties (css, (<&>))
+
+-- Always assumes value is out of 100; will normalize to 100 if a value is passed > 100
+-- Passed props reach the <span>.
+bar :: ∀ p i. Int -> Array (HH.IProp HTMLdiv i) -> Array (HH.IProp HTMLspan i) -> HH.HTML p i
```

Comment:
We could make this a `Rational` instead of an `Int`...would be more accurate, but might add more bulkiness than it's worth?

Reply:
More accurate, perhaps, but I'd rather keep with integers 0-100 until we find another need.